[docs] Incorporated suggestions to docs (#196)
openwisp/openwisp2-docs: tutorials/demo.rst

the core features of OpenWISP Cloud.

If you prefer to use your existing firmware,
please refer to the alternative instructions by reading

> :ref:`the alternative firmware instructions

Reviewer: I am not super convinced of this changed proposed by the copywriter, we have other parts of the text where we link the alternative firmware instructions directly, so this would be inconsistent, moreover, what's the point of repeating it?

Let me know what you think, maybe we can rollback this change.

or via `other means available on OpenWrt
<https://openwrt.org/docs/guide-user/installation/generic.flashing>`_.

> Make sure not to keep settings as shown in

Reviewer: I am also not convinced of this one.

What about something like:

```suggestion
Make sure not to keep settings: supply the ....
```

We could remove the rounded parenthesis and add a colon before the clarification.

--------

If you need help or want to request a free 30-day trial of the
full feature set, you can write to the
full feature set, you can write to us via the
`support channels <https://openwisp.org/support.html>`__ or just click
on "Contact support" as indicated in the screenshot below.
on the tab *Contact support* as indicated in the screenshot below.

.. image:: ../images/demo/contact-support.png
    :target: ../_images/contact-support.png

Next steps
----------

- :doc:`Open and/or WPA protected WiFi Access Point SSID
  <./wifi-access-point>`
- :doc:`WiFi Hotspot, Captive Portal (Public WiFi), Social Login
  <./hotspot>`
- :doc:`How to Set Up a Wireless Mesh Network

>   <./mesh>`

Reviewer: I don't think this removal was intentional. Let's not remove this.
I want to add the WPA Enterprise tutorial here too.
